### PR TITLE
Capitalize SQL language

### PR DESCRIPTION
**Pull request by diana-macrometa**
Took a pass through the Examples folder to start capitalizing the SQL words to bring them in line with preferred style.

### CODE REVIEW

File: docs/cep/examples/data-pipelines.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ CREATE STREAM HumidityStream (roomNo string, humidity double);
 -- Join latest `temperature` and `humidity` events arriving within 1 minute for each `roomNo`.
 INSERT INTO TemperatureHumidityStream
 SELECT t.roomNo, t.temperature, h.humidity
-FROM TemperatureStream window unique:time(roomNo, 1 min) as t
-    join HumidityStream window unique:time(roomNo, 1 min) as h
+FROM TemperatureStream window unique:time(roomNo, 1 min) AS t
+    join HumidityStream window unique:time(roomNo, 1 min) AS h
     on t.roomNo == h.roomNo;
```
Comment (StoyanTc) [inline docs/cep/examples/data-pipelines.md]: `join` and `on` are not fixed.

File: docs/cep/examples/null-example.md [hunk 2]
```diff
@@ -28,24 +28,24 @@ CREATE TABLE GLOBAL ProductInfoTable (item string, discount double);
...
 -- Check if streams attributes `t.discount` and `s.item` are `null`.
-       t.discount is null as isTDiscountNull,
-       s.item is null as isSItemNull
-from ProductInputStream as s
-    left outer join ProductInfoTable as t
+       t.discount is null AS isTDiscountNull,
+       s.item is null AS isSItemNull
+FROM ProductInputStream AS s
+    left outer join ProductInfoTable AS t
     on s.item == t.item;
```
Comment (StoyanTc) [inline docs/cep/examples/null-example.md]: `left outer join` and `on` have remained in small letters.

File: docs/cep/examples/patterns-and-trends.md [hunk 3]
```diff
@@ -119,11 +119,11 @@ CREATE SINK RegulatorActionStream WITH (type='log') (roomNo int, action string);
...
+INSERT INTO RegulatorActionStream
+SELECT e1.roomNo,
 -- Checks whether pattern triggered due to removal of room key.
-    ifThenElse( e2 is null, 'none', 'stop' ) as action
-from every e1=RegulatorStateChangeStream[ action == 'on' ]
+    ifThenElse( e2 is null, 'none', 'stop' ) AS action
+FROM every e1=RegulatorStateChangeStream[ action == 'on' ]
      -> e2=RoomKeyStream
             [ e1.roomNo == roomNo and action == 'removed' ]
         or e3=RegulatorStateChangeStream
```
Comment (StoyanTc) [inline docs/cep/examples/patterns-and-trends.md]: `and`, `or`

File: docs/cep/examples/patterns-and-trends.md [hunk 4]
```diff
@@ -169,9 +169,9 @@ CREATE SINK RoomTemperatureAlertStream WITH (type='log') (roomNo int);
 
 
 -- Alerts if no temperature event having a temperature less than what is set in regulator arrives within 5 minutes after switching on the regulator.
-insert into RoomTemperatureAlertStream
-select e1.roomNo as roomNo
-from e1=RegulatorStateChangeStream[action == 'on']
+INSERT INTO RoomTemperatureAlertStream
+SELECT e1.roomNo AS roomNo
+FROM e1=RegulatorStateChangeStream[action == 'on']
      -> not TemperatureStream[e1.roomNo == roomNo and
```
Comment (StoyanTc) [inline docs/cep/examples/patterns-and-trends.md]: `not`, `and`

File: docs/cep/examples/table-and-store-example.md [hunk 5]
```diff
@@ -19,9 +19,9 @@ CREATE TABLE GLOBAL SensorIdInfoTable (sensorId string, roomNo string);
 
 @info(name = 'Join-query')
 -- Selects `sensorId`, `roomNo`, and `temperature` attributes from stream and table, and adds events to `TemperatureLogTable`.
-insert into TemperatureLogTable
-select t.sensorId as sensorId, s.roomNo as roomNo, t.temperature as temperature
-from TemperatureStream as t join SensorIdInfoTable as s
+INSERT INTO TemperatureLogTable
+SELECT t.sensorId AS sensorId, s.roomNo AS roomNo, t.temperature AS temperature
+FROM TemperatureStream AS t join SensorIdInfoTable AS s
      on t.sensorId == s.sensorId;
```
Comment (StoyanTc) [inline docs/cep/examples/table-and-store-example.md]: `join`, `on`

**Review by jamesm-macrometa — APPROVED** [review]

**Review by StoyanTc — APPROVED** [review]

**Comment by diana-macrometa** [comment]
Thanks, @StoyanTc! I'll get these fixed and then merge.

**Pull request merged**